extensions/binding/org.eclipse.smarthome.binding.hue/src/main/java/org/eclipse/smarthome/binding/hue/handler/HueLightHandler.java

```diff
@@ -309,4 +332,58 @@ public void onLightAdded(HueBridge bridge, FullLight light) {
         }
     }
 
+    /**
+     * Schedules restoration of the alert item state to {@link LightStateConverter#ALERT_MODE_NONE} after a given time.<br>
+     * Based on the initial command:
+     * <ul>
+     * <li>For {@link LightStateConverter#ALERT_MODE_SELECT} restoration will be triggered after <strong>2
+     * seconds</strong>.
+     * <li>For {@link LightStateConverter#ALERT_MODE_LONG_SELECT} restoration will be triggered after <strong>15
+     * seconds</strong>.
+     * </ul>
+     * 
+     * @param channelUID The {@link ChannelUID} which received the initial command
+     * @param command The {@link Command} sent to the item
+     */
+    private void scheduleAlertStateRestore(final ChannelUID channelUID, Command command) {
+        int delay = getAlertDuration(command);
+
+        if (delay > 0) {
+            scheduler.schedule(new Runnable() {
```

Comment:
One thing that probably should also be added is a state update within `onLightStateChanged()`. If the user e.g. set the alert/effect mode in his Philips app, this should be reflected here. Especially, if he turns of an alert - in this case you would have to cancel your scheduled job again. On any "NONE", the scheduler must be cancelled.